### PR TITLE
build update for bpmn

### PR DESCRIPTION
**Pull request by hilak**

### CODE REVIEW

**Comment by hilak** [comment]
@tan-tan-kanarek  please review. Thank you.

File: generator/sources/bpmn/build.xml [hunk 1]
```diff
@@ -36,43 +36,36 @@
...
+	 		<replacefilter token="@DB1_HOST@" value="${activiti.db.hostname}" />
+	 		<replacefilter token="@DB1_PORT@" value="${activiti.db.port}" />
+	 		<replacefilter token="@DB1_USER@" value="${activiti.db.username}" />
+	 		<replacefilter token="@DB1_PASS@" value="${activiti.db.password}" /> 
+	 	</replace>
+	</target>
 
+	<!-- replace with exec task for mvn - skip tests -->
 	<target name="jar" depends="install-contrib">
 		<if>
```
Comment (tan-tan-kanarek) [inline generator/sources/bpmn/build.xml]: Seems like the entire IF is redundant now.

File: generator/sources/bpmn/build.xml [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 	<property environment="env" />
 	<property name="java.lib.path" value="${basedir}/../pojo" />
-	<property name="jar.lib.path" value="${basedir}/lib" />
-	<property name="jar.filename" value="${jar.lib.path}/KalturaClientLib.jar" />
+	<property name="jar.lib.path" value="${basedir}/../pojo/target" />
+	<property name="jar.filename" value="${jar.lib.path}/KalturaClient-3.3.1.jar" /><!-- change JAR name -->
```
Comment (tan-tan-kanarek) [inline generator/sources/bpmn/build.xml]: You can externalize the version as a different variable.

File: generator/sources/bpmn/build.xml [hunk 3]
```diff
@@ -36,43 +36,36 @@
 			</classpath>
 		</taskdef>
 	</target>
+	
+	<!-- replace with an task - replace -->
+	<target name="replace-tokens">
```
Comment (tan-tan-kanarek) [inline generator/sources/bpmn/build.xml]: I would check if ${basedir}/deploy/src/activiti.cfg.xml is not already exists first.

**Comment by tan-tan-kanarek** [comment]
@assafb, please review.

**Comment by hilak** [comment]
@rkreich  Please review

**Comment by hilak** [comment]
According to @rkreich  and @tan-tan-kanarek  there is nothing more to change in the build.xml. I will close this pull request and commit changes to the next branch.

**Pull request closed without merging**